truss/templates/control/control/endpoints.py

```diff
@@ -23,7 +24,10 @@ def proxy(path):
 
     # Wait a bit for inference server to start
     for attempt in Retrying(
-        retry=retry_if_exception_type(ConnectionError),
+        retry=(
+            retry_if_exception_type(ConnectionError)
+            | retry_if_exception_type(ModelNotReady)
+        ),
         stop=stop_after_attempt(INFERENCE_SERVER_START_WAIT_SECS),
```

Comment:
Good point. I think the tradeoff is that for some error cases it may increase the time to return an error. Let's start with this and then we can tune. This will not completely eliminate the issue, models can take any amount of time to load. In those cases it's fine to return the 503 error. Idea is that we retry a bit for the common simple cases.